### ISSUE TITLE
min-block-size and max-block-size are not recognized as valid css properties

### ISSUE THREAD

**Issue by skorasaurus** [validator/validator#1187]
URL being validated or code to reproduce error:
The CSS at https://codepen.io/skorasaurus/pen/gOWmYYO

```
Property max-block-size doesn't exist : 
Property min-block-size doesn't exist : 
```

Both min-block-size and max-block-size are not recognized CSS properties in the validator at https://jigsaw.w3.org/css-validator/validator

Both properties are supported in stable versions of Edge, Chrome, Firefox, and Safari as of this writing. 

ref: https://developer.mozilla.org/en-US/docs/Web/CSS/min-block-size
https://developer.mozilla.org/en-US/docs/Web/CSS/max-block-size

https://drafts.csswg.org/css-logical/#propdef-min-block-size

**Comment by skorasaurus** [comment]
Apologies, but I think this may be more appropriate at ; so I am filing it there.

**Comment by sideshowbarker** [comment]
Thanks for raising this — I’ve fixed it in the sources, and pushed the fix to https://validator.w3.org/nu/

> unsure if it should be reported here or at the css-validator ? [w3c/css-validator#306](https://github.com/w3c/css-validator/issues/306)

It’s good to report it at both places — the fix I made to the HTML checker is just a workaround, and the real fix needs to be made upstream, to the CSS backend
